### PR TITLE
fix: bump packages, fix core-output, fix eslint

### LINKED ISSUE #37: 'set-output' command is deprecated and will be disabled soon
Imported your actions, but it seems they are still using the deprecated format for output when using it in the workflow it would prompt the warning:

> Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: [https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/)

Think you can review and update? 
nice action btw

### PR DESCRIPTION
**Pull request by sammcj**
Fixes #37 

- Bump package versions to latest stable.
- Bump github actions checkout package to current supported version.
- Update build ('package') command for current ncc version and to correctly include licenses of bundled source code (legal requirement).
- Fix broken eslint config.

@falti

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint index.js",
-    "package": "ncc build index.js -o dist",
+    "package": "ncc build -m index.js --minify --v8-cache",
```
Comment (falti) [inline package.json]: I think the `-m` and `--minify` is redundant.
In the PR description you mention something about including the license but this is missing here?

**Comment by falti** [comment]
Thanks for the PR, please have a look at my comment about the `ncc` command.

**Comment by sammcj** [comment]
Whoops I forgot to commit my latest change that moved to the updated ncc package in devdeps, fixed now @falti 😄

**Comment by falti** [comment]
> Whoops I forgot to commit my latest change that moved to the updated ncc package in devdeps, fixed now @falti 😄

Still the `-m` and the `--minify` are the same option, right? Could you remove one or the other, please? I think `--minify` is more descriptive and therefore we could ditch the `-m`?

**Comment by sammcj** [comment]
Oh right, sorry I didn't see any comments on the code. Removed -m

**Review by falti — APPROVED** [review]

**Pull request merged**